api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -1094,15 +1094,23 @@ def _mix_requested(kwargs, opt):
     def delay(self):
         return self._ctx.delay()
 
+    def _get_location_from_point_if_needed(self, location_or_point: Union[types.Location, types.Point]):
```

Comment:
```suggestion
    def _get_location_from_point_if_needed(self, location_or_point: Union[types.Location, types.Point]) -> types.Location:
```
Let's annotate the return type as well as the args